pkgs/desktops/gnome-3/3.24/core/gdm/default.nix

```diff
@@ -18,17 +19,18 @@ stdenv.mkDerivation rec {
   buildInputs = [ pkgconfig glib itstool libxml2 intltool autoreconfHook
                   accountsservice gnome3.dconf systemd
                   gobjectIntrospection libX11 gtk
-                  libcanberra_gtk3 pam libtool plymouth ];
+                  libcanberra_gtk3 pam libtool plymouth librsvg ];
 
   enableParallelBuilding = true;
 
   # Disable Access Control because our X does not support FamilyServerInterpreted yet
-  patches = [ ./xserver_path.patch
+  patches = [ #./xserver_path.patch  # segc
               ./sessions_dir.patch
               ./gdm-x-session_extra_args.patch
               ./gdm-session-worker_xserver-path.patch
-             # ./disable_x_access_control.patch ./no-dbus-launch.patch
-             # ./libsystemd.patch
+#              ./disable_x_access_control.patch
+#	      ./no-dbus-launch.patch
```

Comment:
AFAICT all patches I commented out are no longer needed. I'll remove the lines and the patch files as well?